aml/org.testeditor.aml.model/pom.xml

```diff
@@ -46,7 +46,7 @@
 						</execution>
 					</executions>
 					<configuration>
-						<failOnValidationError>true</failOnValidationError>
+						<classPathLookupFilter>.*(ecore|genmodel|xcore|xtext).*</classPathLookupFilter>
```

Comment:
if we add fixture to the filter:						
<classPathLookupFilter>.*(ecore|genmodel|xcore|xtext|fixture).*</classPathLookupFilter>

it works also with fixture jars